### PR TITLE
docs: add architecture info & dockerhub url

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description
This is a part of gitpod-io/website#2341

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by kylos101** [comment]
@Siddhant-K-code I assume this is not ready for review yet (because it's in draft) and am going to review Team Workspace. Please add us back once ready, and click the "Ready for review" button. Thank yous!

**Comment by Siddhant-K-code** [comment]
@kylos101 It is good for review now. I will create the follow-up PR with integration of https://github.com/gitpod-io/dazzle/pull/56

**Comment by Siddhant-K-code** [comment]
@kylos101 It is good for review now. I will create the follow-up PR with integration of https://github.com/gitpod-io/dazzle/pull/56

File: README.md [hunk 1]
```diff
@@ -3,70 +3,83 @@
...
 
 ## 📢 Announcements
 
 We upgraded to support [dazzle v2](https://github.com/gitpod-io/dazzle) and build with GitHub Actions.
 
-### 📷 Images we'll upgrade
+## Summary
+
+- **Operating System**: Ubuntu 20.04.4 LTS
+- **OS Type**: linux
```
Comment (jenting) [inline README.md:17]: nit: Linux

**Review by jenting — COMMENTED** [review]
LGTM

**Comment by jenting** [comment]
CI failed on pre-commit. Could you please check?
Thank you.

**Comment by Siddhant-K-code** [comment]
We should ignore the `Build` action for markdown changes. 🤔

**Review by jenting — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,70 +3,88 @@
...
+- **OS Type**: Linux
+- **Architecture**: x86_64
+
+## Images
+
+By default, Gitpod uses a standard Docker Image called [`workspace-full`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L23) as the foundation for workspaces.
+
+Workspaces started based on this default image come pre-installed with Docker, Nix, Go, Java, Node.js, C/C++, Python, Ruby, Rust & PHP as well as tools such as Homebrew, Tailscale, Nginx & several more.
+
+If this image does not include the tools you need for your repository, we recommend you use [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3).
```
Comment (Furisto) [inline README.md:26]:
```suggestion
If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
```

File: README.md [hunk 3]
```diff
@@ -3,70 +3,88 @@
...
+OR, [Bring your Dockerfile](https://www.gitpod.io/blog/docker-in-gitpod).
```
Comment (Furisto) [inline README.md:27]: I would not recommend referencing the blog post as it is 4 years old at this point, was written before dazzle and contradicts what is written here (it uses workspace-full)

Comment (Siddhant-K-code) [reply README.md:27]: ok, I will update our docs page.

```suggestion
OR, [Bring your Dockerfile](https://www.gitpod.io/docs/configure/workspaces/workspace-image#configure-a-public-docker-image).
```

File: README.md [hunk 4]
```diff
@@ -3,70 +3,91 @@
...
+
+Workspaces started based on this default image come pre-installed with Docker, Nix, Go, Java, Node.js, C/C++, Python, Ruby, Rust & PHP as well as tools such as Homebrew, Tailscale, Nginx & several more.
+
+If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and,
+
+customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
+
+OR, [Bring your Dockerfile](https://www.gitpod.io/docs/configure/workspaces/workspace-image#configure-a-public-docker-image).
+
+According to your repository. You can refer to this [documentation to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
```
Comment (Furisto) [inline README.md:32]: This is now superfluous.

File: README.md [hunk 5]
```diff
@@ -3,70 +3,91 @@
...
+
+## Images
+
+By default, Gitpod uses a standard Docker Image called [`workspace-full`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L23) as the foundation for workspaces.
+
+Workspaces started based on this default image come pre-installed with Docker, Nix, Go, Java, Node.js, C/C++, Python, Ruby, Rust & PHP as well as tools such as Homebrew, Tailscale, Nginx & several more.
+
+If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and,
+
+customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
```
Comment (Furisto) [inline README.md:28]:
```suggestion
If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
```

**Comment by Siddhant-K-code** [comment]
Can we disable markdownlint? it breaks the flow of paragraph and wants us to limit a paragraph to 200 characters only 🤔

File: README.md (hunk 4 above)
Comment (Siddhant-K-code) [inline README.md:30]:
```suggestion
```

**Comment by loujaybee** [comment]
> @loujaybee if you have time, could you please help review this? Thank you. - @jenting 

If you folks are happy, please proceed! It's my fault, when @Siddhant-K-code asked me to review, I assigned to show that I would look, but I haven't yet had time — if you're happy, please proceed, don't let me block you! 

Will remove my assignee! 🙏

File: README.md [hunk 6]
```diff
@@ -3,70 +3,88 @@
...
+
+## Images
+
+By default, Gitpod uses a standard Docker Image called [`workspace-full`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L23) as the foundation for workspaces.
+
+Workspaces started based on this default image come pre-installed with Docker, Nix, Go, Java, Node.js, C/C++, Python, Ruby, Rust & PHP as well as tools such as Homebrew, Tailscale, Nginx & several more.
+
+If this image does not include the tools you need for your repository, we recommend you start with [`workspace-base`](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml#L3) and customize it according to your needs. You can refer to this [document to setup a custom docker image](https://www.gitpod.io/docs/configure/workspaces/workspace-image).
+
+OR, [Bring your Dockerfile](https://www.gitpod.io/docs/configure/workspaces/workspace-image#configure-a-public-docker-image).
```
Comment (Furisto) [inline README.md:28]: This points to the same document as the one in the sentence above. Can this be removed?

**Review by Furisto — APPROVED** [review]

**Pull request merged**